src/parser/hunter/survival/modules/Abilities.js

```diff
@@ -113,6 +113,7 @@ class Abilities extends CoreAbilities {
       },
       { //WFI talent here first so that's the icon shown in the timeline - it has no other effect.
         spell: [SPELLS.WILDFIRE_INFUSION_TALENT, SPELLS.VOLATILE_BOMB_WFI, SPELLS.PHEROMONE_BOMB_WFI, SPELLS.SHRAPNEL_BOMB_WFI],
+        shownSpell: SPELLS.WILDFIRE_INFUSION_TALENT,
```

Comment:
The first spell in the spell array is shown in the timeline, so then this doesn't do anything?